src/components/action.ts

```diff
@@ -75,6 +119,24 @@ export class Action extends LitElement {
         padding: 10px;
         margin-top: 20px;
       }
+      #raise-button,
```

Comment:
Should add a class to these buttons. `action-button`. So we don't need to call the IDs for each one.